packages/constants/src/assets.test.ts

```diff
@@ -2,7 +2,24 @@ import { describe, expect, it } from 'vitest';
 import { assetPatterns, RegexMatcher } from './assets';
 
 describe('assetPatterns', () => {
-  describe('lpNftPattern', () => {
+  describe('auctionNft', () => {
```

Comment:
This change isn't strictly needed by this PR, but it will be needed shortly when I implement the `auctions` RPC method